README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Name
+
+Hello, my name is **Nadiah Wong**! 
+
+
+## Who am I?
+
+I was born on 6 July 1997 in sunny Singapore
+This is what I look like: ![*Premature graduation photos because we probably would not be back together again soon :-(*](/Users/nadiah/Desktop/grad photos/DSC03848-2)
```

Comment:
I'm not sure if you are using R locally or on the server. If you are using the server, I think the picture didn't work because it was not uploaded onto your working space on R.

Reply:
Thanks for pointing that out, Vivian! I just edited and the photo should work. Let me know if it does not!